drift_dev/lib/src/analysis/options.dart

```diff
@@ -61,6 +61,10 @@ class DriftOptions {
   @JsonKey(name: 'generate_connect_constructor', defaultValue: false)
   final bool generateConnectConstructor;
 
+  /// Generate managers to assist with common database operations.
+  @JsonKey(name: 'generate_manager', defaultValue: true)
+  final bool generateManager;
```

Comment:
This option should also be mentioned in `docs/pages/docs/Generation options/index.md`.

Reply:
Done